### ISSUE TITLE
Why still no IGTV uploads?

### ISSUE THREAD

**Issue by AKAlex92** [ping/instagram_private_api#343]
## Please follow the guide below

- Issues submitted without this template format will be **ignored**.
- Read the questions **carefully** and answer completely.
- Do not post screenshots of error messages or code.
- Put an `x` into all the boxes [ ] relevant to your issue (==> [x] *no* spaces).
- Use the *Preview* tab to see how your issue will actually look like.
- Issues about reverse engineering is out of scope and will be closed without response.
- Any mention of spam-like actions or spam-related tools/libs/etc is strictly **not allowed**.

---

### Before submitting an issue, make sure you have:
- [x] Updated to the lastest version v1.6.0
- [x] Read the [README](https://github.com/ping/instagram_private_api/blob/master/README.md) and [docs](https://instagram-private-api.readthedocs.io/en/latest/)
- [x] [Searched](https://github.com/ping/instagram_private_api/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] Reviewed the sample code in [tests](https://github.com/ping/instagram_private_api/tree/master/tests) and [examples](https://github.com/ping/instagram_private_api/tree/master/examples)

### Which client are you using?
- [x] app (``instagram_private_api/``)
- [ ] web (``instagram_web_api/``)

---

### Describe your Feature Request:

Thank you for your work, i really enjoy that repo but..
Why not develop a post_igtv_video method? Why aren't IGTV video uploads supported?
I only can post video with duration <= 60 seconds.
I tried also with reels = True but..the aspect ratio wasn't compatible at all.
Thank you for your time.

**Comment by Lemmedie** [comment]
How can u upload videos ? the uploading endpoint has been closed

**Comment by AKAlex92** [comment]
> How can u upload videos ? the uploading endpoint has been closed

To be honest I didn't even reach to understand if the endpoints worked, I was just blocked by those errors before the uploading began.
